src/ctia/bundle/routes.clj

```diff
@@ -3,17 +3,53 @@
   (:require
    [compojure.api.sweet :refer :all]
    [ctia.bundle
-    [core :refer [bundle-size
-                  bundle-max-size
-                  import-bundle]]
+    [core :refer [bundle-max-size
+                  bundle-size
+                  import-bundle
+                  export-bundle]]
     [schemas :refer [BundleImportResult]]]
-   [ctia.schemas.core :refer [NewBundle]]
+   [ctia.schemas.core :refer [Bundle NewBundle]]
    [ring.util.http-response :refer :all]
    [schema.core :as s]))
 
+(s/defschema BundleExportQuery
+  {:ids [s/Str]
+   (s/optional-key :include_related_entities) s/Bool})
+
 (defroutes bundle-routes
   (context "/bundle" []
            :tags ["Bundle"]
+           (GET "/export" []
+                :return Bundle
+                :header-params [{Authorization :- (s/maybe s/Str) nil}]
+                :query [q BundleExportQuery]
+                :summary "Export a record with its local relationships"
+                :capabilities #{:read-actor
+                                :read-attack-pattern
+                                :read-campaign
+                                :read-coa
+                                :read-exploit-target
+                                :read-feedback
+                                :read-incident
+                                :read-indicator
+                                :list-indicators
+                                :read-judgement
+                                :list-judgements
+                                :read-malware
+                                :read-relationship
+                                :list-relationships
+                                :read-sighting
+                                :list-sightings
+                                :read-tool
+                                :read-verdict}
```

Comment:
Why the `:list-` capability is only required for indicators, judgements, relationships and sightings? All entities can have a relationship between them.